### DISCUSSION TITLE
[✨]Add focus as qwikify eagerness option

### DISCUSSION THREAD

**Discussion by rolanday** [QwikDev/qwik-evolution#144] [Proposals For Qwik]
### Is your feature request related to a problem?

Eagerness options for qwikify are hover, idle, load, and visible. If choosing hover, presumably because only want to activate component if user interacts with it, does not address the case where user change focus to control via keyboard tabbed input. This prevents usage if accessibly is a requirement.

### Describe the solution you'd like

Pease add focus, so item is activated when tabbed to -- will need to be or-able with hover or, overload hover to include if receives focus via keyboard.

### Describe alternatives you've considered

Use more aggressive visibility option (such as visible), but this comes at cost of potentially paying the download price for inactive component.

**Comment by wmertens** [comment]
Indeed, I'd say that `focus` should be tracked together with `hover`.

**Comment by gioboa** [comment]
We moved this issue to `qwik-evolution` repo to create a RFC discussion for this.
Here is our [Qwik RFC process](https://github.com/QwikDev/qwik-evolution?tab=readme-ov-file#qwik-rfc-process) thanks.
